client/fieldregistry.go

```diff
@@ -59,6 +59,8 @@ type fieldRegistry struct {
 	ObjectSID          *Field `ldap:"objectSid"`
 	OrganizationalUnit *Field `ldap:"ou"`
 	PasswordLastSet    *Field `ldap:"passwordLastSet"`
+	RcafPassword       *Field `ldap:"rcafPassword"`
```

Comment:
nit: should these be "RCAFPassword/Attributes"?

Reply:
Done!